src/components/test/buttons-test.js

```diff
@@ -20,6 +21,13 @@ function addCommonTests({ componentName, createComponentFn, withIcon = true }) {
       });
     }
 
+    it('passes along a `ref` to the button element', () => {
+      const buttonRef = createRef();
+      createComponentFn({ ref: buttonRef });
+
+      assert.isTrue(buttonRef.current instanceof Node);
```

Comment:
This would fail if we don't forward the ref because the object would be a Preact component, not a Node.